src/Demos/HSP.lagda

```diff
@@ -123,13 +100,8 @@ See the \href{https://agda.readthedocs.io/en/v2.6.1/tools/command-line-options.h
 \end{itemize}
 \fi
 
-The {\footnotesize \AgdaKeyword{OPTIONS}} pragma is usually followed by the start of a module and a list of
-import directives. For example, the present module (\DemosHSPlagda)
-\ifshort includes the following imports.
-\else
-begins as follows.
-\fi
-
+We make use of a variety of standard definitions that can be found in Agda's standard library.
```

Comment:
I'd rather not use "standard" as an adjective twice in one sentence.  It just sounds odd to me.  Instead, the second "standard" could be part of the name "Agda Standard Library."

Reply:
Or even better, drop the first 'standard'!

Reply:
Great idea! :bulb: :100: :smile: